src/lib/command_line.ml

```diff
@@ -125,9 +128,9 @@ let pipeline ~biokepi_machine ?work_directory =
     (`With_varscan with_varscan)
     (`With_somaticsniper with_somaticsniper)
     (`Bedfile bedfile)
-    (`Normal_json normal_json_file)
-    (`Tumor_json tumor_json_file)
-    (`Rna_json rna_json_file)
+    (`Normal_json normal_json_files)
```

Comment:
Is there documentation on what all these pipeline tags mean? At the very least, could you comment them?